src/phaser-game/scenes/FortNerf.ts

```diff
@@ -1,168 +1,198 @@
+      bullet = this.physics.add.sprite(x, y, 'bullet');
+      bullet.onCollide = true;
+      bullet.body.onCollide = true;
+      if (direction === 'right') {
+        bullet.setVelocityX(BULLET_MOVEMENT);
+      } else if (direction === 'left') {
+        bullet.setVelocityX(-BULLET_MOVEMENT);
+      } else if (direction === 'down') {
+        bullet.setVelocityY(BULLET_MOVEMENT);
+      } else if (direction === 'up') {
+        bullet.setVelocityY(-BULLET_MOVEMENT);
+      }
+    };
     // player animations
+    // movement animation function
+    const createMoveAnimations = (
+      keyFrame: string,
+      startFrame: number,
+      endFrame: number,
+    ) => {
+      this.anims.create({
+        key: keyFrame,
+        frames: this.anims.generateFrameNumbers('player', {
+          start: startFrame,
+          end: endFrame,
+        }),
+        frameRate: 10,
+        repeat: -1,
+      });
+    };
+    // still animation function
+    const createStillAnimation = (
+      keyFrame: string,
+      frameNumber: number,
+    ) => {
+      this.anims.create({
+        key: keyFrame,
+        frames: [{ key: 'player', frame: frameNumber }],
+        frameRate: 20,
+      });
+    };
     // movement animations
-    this.anims.create({
-      key: 'left',
-      frames: this.anims.generateFrameNumbers('player', {
-        start: 3,
-        end: 5,
-      }),
-      frameRate: 10,
-      repeat: -1,
-    });
-
-    this.anims.create({
-      key: 'right',
-      frames: this.anims.generateFrameNumbers('player', {
-        start: 6,
-        end: 8,
-      }),
-      frameRate: 10,
-      repeat: -1,
-    });
-
-    this.anims.create({
-      key: 'down',
-      frames: this.anims.generateFrameNumbers('player', {
-        start: 0,
-        end: 2,
-      }),
-      frameRate: 10,
-      repeat: -1,
-    });
-
-    this.anims.create({
-      key: 'up',
-      frames: this.anims.generateFrameNumbers('player', {
-        start: 9,
-        end: 11,
-      }),
-      frameRate: 10,
-      repeat: -1,
-    });
+    createMoveAnimations('left', 3, 5);
+    createMoveAnimations('right', 6, 8);
+    createMoveAnimations('down', 0, 2);
+    createMoveAnimations('up', 9, 11);
 
     // still animations
-    this.anims.create({
-      key: 'downstill',
-      frames: [{ key: 'player', frame: 1 }],
-      frameRate: 20,
-    });
-
-    this.anims.create({
-      key: 'upstill',
-      frames: [{ key: 'player', frame: 10 }],
-      frameRate: 20,
-    });
-
-    this.anims.create({
-      key: 'leftstill',
-      frames: [{ key: 'player', frame: 4 }],
-      frameRate: 20,
-    });
-
-    this.anims.create({
-      key: 'rightstill',
-      frames: [{ key: 'player', frame: 7 }],
-      frameRate: 20,
+    createStillAnimation('leftstill', 4);
+    createStillAnimation('rightstill', 7);
+    createStillAnimation('downstill', 1);
+    createStillAnimation('upstill', 10);
+
+    // bullet animations
+
```

Comment:
Noticing a little inconsistency with extra lines (or the absence thereof) between functions and variable blocks. Just try to tighten that up/be more consistent going forward.